.pep8speaks.yml

```
@@ -0,0 +1,8 @@
+scanner:
+  diff_only: False  # If False, the entire file touched by the Pull Request is scanned for errors. If True, only the diff is scanned.
+  linter: flake8
+
+flake8:
+  max-line-length: 88  # flake8-bugbear uses 80*1.1
+  ignore:
+    - W503
```

Add new line at end of file.
```suggestion
    - W503

```